Header header
float64 a       # a = velocity(rpm/s)
float64 delta   # delta = angle(rad)